app/controllers/api/vcs.php

```diff
@@ -884,7 +1019,7 @@
         $repository['organization'] = $installation->getAttribute('organization', '');
         $repository['provider'] = $installation->getAttribute('provider', '');
 
-        $response->dynamic(new Document($repository), Response::MODEL_PROVIDER_REPOSITORY);
+        $response->dynamic(new Document($repository), Response::MODEL_RUNTIME_PROVIDER_REPOSITORY);
```

Comment:
This endpoint doesnt seem to be setting runtime. Let's check with Arman to understand when this is beign used, so we can decide if we want type parameter here too.

Reply:
Can we use abstract providerRepository model instead?